source/common/http/filter_manager.h

```diff
@@ -36,6 +36,30 @@ class DownstreamFilterManager;
 
 struct ActiveStreamFilterBase;
 
+constexpr absl::string_view FS_LOCAL_REPLAY_KEY = "envoy.filters.listener.local_replay_owner";
+class LocalReplyOwnerType : public StreamInfo::FilterState::Object {
+public:
+  LocalReplyOwnerType(const std::string& filter_config_name, const std::string& details)
+      : filter_config_name(filter_config_name), details(details) {}
+
+  ProtobufTypes::MessagePtr serializeAsProto() const override {
+    auto message = std::make_unique<ProtobufWkt::Struct>();
+    auto& fields = *message->mutable_fields();
+    *fields["filter_config_name"].mutable_string_value() = filter_config_name;
+    *fields["details"].mutable_string_value() = details;
+
+    return message;
+  }
+
+  absl::optional<std::string> serializeAsString() const override {
+    return filter_config_name + "|" + details;
+  }
+
+private:
+  const std::string filter_config_name;
+  const std::string details;
```

Comment:
Hmmm, I doubt if this detail is necessary here? I think we have record this detail in the stream info?

Reply:
yes, it is available as attribute

Reply:
If wasm filter connot comsume this value, then it would be better to enhance the ABI to expose it. So, just let's remove it here.